### DISCUSSION TITLE
Display "choices" - use or copy from Interface?

### DISCUSSION THREAD

**Discussion by takuy** [directus/directus#17327] [Feature Requests]
I think there should be an option in "Display" options for a field, to simply "Use Interface choices" for things like Dropdowns. I've already configured my "labels" and "values" in the Interface options. Should just have the option of using those for Display options as well.

**Comment by shawnweeks** [comment]
This was very confusing to me as well. I haven't found anything that actually said why you have to put your choices under Interfaces so that you can select them and under Display so you can format them. You don't even have to put them under Display if your not formatting them. In fact on the Display tab you don't even have to make the text match, just the value.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by MarvinXu** [comment]
Why do I have to create "choices" in both interface and display?

**Reply by rijkvanzanten** [reply]
Because the two aren't linked. You can use the interface without the display, and the display without the interface.
